doc/include/api_extcproc.inc

```diff
@@ -80,34 +80,32 @@ It returns a boolean value, [const true] if the type is known and
 [call [cmd ::critcl::argtype] [arg name] [arg body] [opt [arg ctype]] [opt [arg ctypefun]]]
 
 This command defines the argument type [arg name], and associates it
-with the C code doing the conversion ([arg body]) from Tcl to C
+with the C code doing the conversion ([arg body]) from Tcl to C.
 
-The C type of the variable to hold the conversion result is
-[arg ctype] and the type of the function argument itself is
-[arg ctypefun].  Both types default to [arg name] if they are not
-specified (or the empty string).
+[arg ctype] is the C type of the variable to hold the conversion result 
+and [arg ctypefun] is the type of the function argument itself.
+Both types default to [arg name] if they are the empty string or are not
+provided.
 
-[para] If [arg name] is declared already an error will be thrown.
+[para] If [arg name] is already declared an error is thrown.
 
-[para] The [arg body]'s responsibility is the conversion of a
-command's Tcl_Obj* argument into a C value for the underlying
-function and its storage in a helper variable.
+[para] [arg body] is a function that converts a Tcl_Obj* into a C value which
+is stored in a helper variable in the underlying function.
 
-[para] The C code of [arg body] is guaranteed to be called inside of a
-separate C code block (thus allowing the use of local variables) which
-has access to the following environment:
+[para] [arg body] is called inside its own code block to isolate local
+variables, and the following items are in scope:
 
 [list_begin definitions]
-[def [var interp]] A Tcl_Interp* typed C variable referencing the
+[def [var interp]] A variable of type [const Tcl_Interp*] which is the
                    interpreter the code is running in.
-[def [const @@]] A placeholder for the Tcl_Obj*-valued C expression
-     	    	 providing the value of the argument to convert.
+[def [const @@]] A placeholder for an expression that evaluates to the [
+     	    	 const Tcl_Obj*] to convert 
```

Comment:
Trailing whitespace. Command broken across lines.

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).